packages/babel-types/src/validators/isReferenced.js

```diff
@@ -54,16 +54,18 @@ export default function isReferenced(
       if (parent.params.includes(node)) {
         return false;
       }
-    // Fall-through to next case clause to check whether the node is the method's name.
+    // fall through
 
     // yes: { [NODE]: "" }
     // no: { NODE: "" }
     // depends: { NODE }
     // depends: { key: NODE }
+    // fall through
```

Comment:
@kaicataldo The comment on line 57 is not enough. Could https://github.com/eslint/eslint/issues/10608 be reopened? :pray: :sweat_smile:

Reply:
I wonder if we should allow it to be either the first or last, but not the middle? I know it's unlikely, but I think it would be good to avoid something like this triggering it:

```js
/* eslint no-fallthrough: ["error", { "commentPattern": "fall-through[\\s\\w]*not allowed" }] */

switch(foo) {
    case 1:
        doSomething();
        // We don't want to fall through here
        // because fall-through is not allowed,
        // oh look, a run on sentence.

    default:
        doSomething();
}

```

Reply:
Doesn't this code already have that problem?

```js
/* eslint no-fallthrough: ["error", { "commentPattern": "fall-through[\\s\\w]*not allowed" }] */

switch(foo) {
    case 1:
        doSomething();
        // We don't want to fall through here because fall-through is not allowed.

    default:
        doSomething();
}
```

Reply:
Yeah, it would. Might just be me, but the grouped line comments seem more error prone. It probably is the same thing in practice, though.